### ISSUE TITLE
Screen reader does not read pre/suffix

### ISSUE THREAD

**Issue by eivindland** [microsoft/fluentui#13758]
## Environment Information

- **Package version(s)**: 7.110.2
- **Browser and OS versions**: Safari, Firefox

### Describe the issue:

When you use prefix and/or suffix on a TextField, the screen reader software does not read the pre/suffix value. For instance if the you have a textfield with the label "Weight" and the suffix "grams", the screen reader will simply say "Weight" and ignore "grams", forcing you use additional labelling.

#### Expected behavior:

The input field contains a reference to the pre/suffix div by using something like:
 <input aria-describedby='my_suffix1'>...<div id='my_suffix1'>grams</div>

**Comment by paulgildea** [comment]
@eivindland is this a bug with the current `TextField` issue or is this a feature request for the `TextField` component?

**Comment by eivindland** [comment]
@paulgildea Well, we have internally marked is as a bug with the current TextField since it easy for developers to use it in such a way that it does not been the accessiblity requirements. (It is however possible to meet the requirements by adding good labelling manually, but I think it would be a much better solution to make the screen reader say the pre/sufffix out loud automatically.)

One example: If we have a form containing TextFields with units as suffix (inch, feet, meter, centimeter), a blind user is not told what units we are asking when entering text. The developer has to label the TextField "Length in ft" or something similar.

**Comment by paulgildea** [comment]
@eivindland - Thanks for the clarification. 

I don't know if we can provide a reliable solution for TextField if units are embedded within the text of TextField were we can automatically provide that context to the screen reader.

I like the idea - but I don't believe we can achieve a predictable solution at this time. 

I'm going to close this issue since it's blocking because while frustrating, manual labeling can achieve the right outcome.
